### ISSUE TITLE
Publish the latest version on maven

### ISSUE THREAD

**Issue by atapin** [onflow/flow-jvm-sdk#13]
We only see v0.1.1 on public maven. Can you guys publish the latest version?

**Comment by psiemens** [comment]
Yep, we're just getting our pipeline set up -- sorry for the delay, @atapin!

@briandilley and his team have been maintaining this repository (🙏 ), so I can confirm that their fork is the one to use for now. We'll get everything over to the `com.onflow` org next week.

**Comment by atapin** [comment]
That’s perfectly fine. Thanks for letting me know.
